osquery/config/plugins/tls.h

```diff
@@ -31,6 +31,9 @@ class TLSConfigPlugin : public ConfigPlugin,
 
  private:
   friend class TLSConfigTests;
+  Status updateDelayPeriod(Status s);
```

Comment:
nit, reorganize these a bit, there's a section right above, marked `protected` which holds private member variables. Place them there and add docblocks, `///` is OK for one-liners.